tests/push/gitlab_pull_request_test.py

```diff
@@ -2,6 +2,7 @@
 
 import json
 import subprocess
+import urllib
```

Comment:
importing `urllib` imports nothing -- this needs to actually import the submodules (right now this is depending on import side-effects -- I'm surprised mypy doesn't catch this)

Reply:
Fixed.